src/user/user.controller.ts

```diff
@@ -12,6 +13,12 @@ export class UserController extends Controller {
 
         router.get('/:id', this.getUser);
         router.put('/:id', this.updateUser);
+        router.patch('/activate/:id', this.activateUser);
```

Comment:
No.
We should generate very long random string that will be used to activate account! So, we need to create another database table with pending verification requests that are bound to user.

Also password reset request: email should not be passed as params. This is secret data. We need to pass it thru body.

Also those endpoints need to be moved to auth controller / service. They are more auth than user related